### DISCUSSION TITLE
Do these gaps in floating windows have their own highlight groups?

### DISCUSSION THREAD

**Discussion by tesarlukas** [NvChad/NvChad#3050] [Q&A] [answered]
Hi, I'm trying to edit my Neovim UI a bit and I wonder if the gaps on many of these windows have their respective highlight groups. 

For example here in floating nvim-tree, there's a small gap on both top and bottom of the window that have the same colour as the nvim's background:
![image](https://github.com/user-attachments/assets/fa77df33-ae29-4d3a-be2c-44f594534c21)

I tried to change every highlight group that was in nvim-tree docs and none of them has changed the colour of that stripe.

Grapple has something similar:
![image](https://github.com/user-attachments/assets/f5d510ab-8949-47d5-85b6-5912031fee4a)

And even floating LSP windows: 
![image](https://github.com/user-attachments/assets/1d66a060-d23d-49ad-9fe9-1c0317629940)

One fix that came to my mind was to make the `darker_black` the same colour as the nvim's background, but I kinda like having these darker tones on the floating windows.

**Comment by siduck** [comment, marked as answer]
those are the hlgroups of the border itself, if i were you i'd make the border's bg same as window's bg and remove the fg color from border

or remove the bg color from the window
